util/unittests/test_html.py

```diff
@@ -123,7 +123,7 @@ def test02_isalistofdictionaries(self):
                          ('Testing isalistofdictionaries output. '
                           'Assert false 1'))
 
-    def test03_isalistofdictionaries(self):
+    def test_isalistofdictionaries(self):

```

Comment:
__[flake8]__
* [F811] redefinition of unused 'test_isalistofdictionaries' from line 113

<!-- Comment by [SideCI](https://sideci.com) -->